jolt-core/src/test/resources/json/utils/joltUtils-store-remove-compact.json

```diff
@@ -0,0 +1,271 @@
+[
+    {
+        "source": {
```

Comment:
add a "testDescription" field to this data structure, aka what is each test getting at?